### PR TITLE
phase 1 changes - typos, destroyxerath, and alphabetical lists

### PR DESCRIPTION
**Pull request by WKWCoding**
I don't know how to test this before I push it. But I think it should work.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -30,9 +30,17 @@ client.on('message', message => {
...
+                break;
+            case "deletexerath":
+                message.channel.send('Xerath Has been permanently deleted. Good Riddance.');
+                break;
+			case "help":
+			    message.channel.send('Try these commands: save, destroy, count, list, alist, and deletexerath.');
+			    break;
 			default:
-				message.channel.send(`I can't do that. Try 'save' or 'destroy', followed by a city name`);
+				message.channel.send(`I can't do that. Try 'save' or 'destroy', followed by a city name.`);
```
Comment (webstersw) [inline src/index.ts]: change this to "try !citysave help"

File: src/commandHelpers.ts [hunk 2]
```diff
@@ -14,20 +14,29 @@ export const handleCity = async (args: string[], message: Message, saved: boolea
...
+            message.channel.send(`The city of ${city} was destroyed. You hate to see it.`)
         }
     }
 };
 
 export const getList = (args: string[], message: Message): void => {
-    const saved = getCityList(true);
-    const destroyed = getCityList(false);
+    const saved = getCityList(true).sort((n1,n2) => n1 - n2);
+    const destroyed = getCityList(false).sort((n1,n2) => n1 - n2);;
```
Comment (webstersw) [inline src/commandHelpers.ts]: Wrong method, add this to getAlphabeticalList

File: src/index.ts [hunk 3]
```diff
@@ -30,9 +30,17 @@ client.on('message', message => {
 			case "list":
 				getList(args, message);
 				break;
-			case "help":		
+            case "alist":
```
Comment (webstersw) [inline src/index.ts]: nit: tabbing

**Review by webstersw — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 4]
```diff
@@ -1,8 +1,9 @@
 import { Client } from 'discord.js';
 import config from './config';
-import { getCount, handleCity, getList } from './commandHelpers';
+import { getCount, handleCity, getList, getAlphabeticalList } from './commandHelpers';
 
 const client = new Client();
+const tok = "Nzg1NjY1OTg0NTUwOTI4Mzg0.X87Kdw.gb3G-4AtTNiAixQdIyIhemuS3kQ";
```
Comment (webstersw) [inline src/index.ts]: token revealed - rekt - use an environment variable

**Review by webstersw — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 4 above)
Comment (WKWCoding) [reply src/index.ts]: check your messages

File: src/commandHelpers.ts [hunk 5]
```diff
@@ -26,8 +26,18 @@ export const getList = (args: string[], message: Message): void => {
     message.channel.send(`Destroyed cities: ${destroyed}`);
 };
 
+export const getAlphabeticalList = (args: string[], message: Message): void => {
+    var saved = JSON.parse(JSON.stringify(getCityArray(true)));
+    saved.sort();
+    var destroyed = JSON.parse(JSON.stringify(getCityArray(false)));
```
Comment (webstersw) [inline src/commandHelpers.ts]: JSON.parse(JSON.stringify(... no

Comment (webstersw) [reply src/commandHelpers.ts]: try one of these
```
// "Spread"
{ ...food }
// "Object.assign"
Object.assign({}, food)
```

**Review by webstersw — APPROVED** [review]

**Pull request merged**